Remove `domain` for testing setup (#498)
nebari-dev/nebari-docs: docs/docs/how-tos/nebari-local.md

Note You will be prompted to enter values for some of the choices above if they are absent from the command line arguments (for example, project name and domain)
:::

:::note

> If you are deploying for testing purposes, omitting the `--domain` option is recommended to prevent issues related to setting up a DNS domain for your deployment. Nebari will return the external IP associated with the Kubernetes cluster's load balancer by default.

Reviewer: ```suggestion
If you are deploying for testing purposes, omitting the `--domain` option is recommended to prevent issues related to setting up a DNS domain for your deployment. By default, at the end of your deployment, Nebari will return the external IP associated with the Kubernetes cluster's load balancer. 
```

Note You will be prompted to enter values for some of the choices above if they are absent from the command line arguments (for example, project name and domain)
:::

> :::note
> If you are deploying locally, DNS is not necessary except for dask dashboard and the notebook
> scheduler extension. Therefore for test deployments it is recommended leave out the `--domain` option.
> :::

Reviewer: ```suggestion
:::note
If you are deploying for testing purposes, omitting the `--domain` option is recommended to prevent issues related to setting up a DNS domain for your deployment. Nebari will return the external IP associated with the Kubernetes cluster's load balancer by default.
:::
```

        tag: sha-b4a2d1e
```

### Disable DNS for local deployment

> If you are deploying locally, DNS is not necessary except for dask dashboard and the notebook

Reviewer: I would move this as a `:::note` under `## Initializing Nebari` and add a optional for `### Setting up a DNS record for the domain` this last bit could also be moved to the DNS section as well
> If you prefer to setup DNS, see the [domain-registry documentation](https://www.nebari.dev/docs/how-tos/domain-registry#what-is-a-dns).

...
```

> To login to the dashboard, you will first need to [login with keycloak](https://www.nebari.dev/docs/how-tos/configuring-keycloak), which is at https://projectname.domain/auth/ 
> (or ip if you are not using a `domain`.

Author: This chunk probably isn't necessary in retrospect.
```suggestion
```

Note the above snippet can be automatically provisioned in your configuration if you provided the `--ssl-cert-email` flag when you ran `nebari init`.

Let's Encrypt heavily rate limits their production endpoint.  In order to avoid throttling, Nebari's traefik deployments will [store certificates in an acme.json file](https://doc.traefik.io/traefik/https/acme/#storage) for the duration of their validity.  Nebari will mount a PVC and save the file on the container at the location `/mnt/acme-certificates/acme.json`.

:::note
In order to refresh the certificate before it is invalidated, you will need to delete the `acme.json` file then restart the Traefik deployment by deleting the existing pod and letting a new one spin up.  This may be necessary if you change the domain name of your Nebari deployment.
:::

### Login and Try Nebari!

To login to the dashboard, you will first need to [login with
keycloak](https://www.nebari.dev/docs/how-tos/configuring-keycloak).

> Assuming you are not using a `domain`, visit https://172.18.1.100/auth/admin/ in your browser and

Reviewer: This IP can change from your docker default subnet, which might vary depending on how the docker daemon was set up. I would suggest following the links provided after your deployment:
```shell
[terraform]: Nebari deployed successfully
Services:
 - argo-workflows -> https://xxx.xx.x.xxx/argo/
 - conda_store -> https://xxx.xx.x.xxx/conda-store/
 - dask_gateway -> https://xxx.xx.x.xxx/gateway/
 - jupyterhub -> https://xxx.xx.x.xxx/
 - keycloak -> https://xxx.xx.x.xxx/auth/
 - monitoring -> https://xxx.xx.x.xxx/monitoring/
Kubernetes kubeconfig located at file:///tmp/NEBARI_KUBECONFIG
Kubecloak master realm username=root password=******
Additional administration docs can be found at https://www.nebari.dev/docs/how-tos/configuring-keycloak
```

It might also show up in the middle deployment in the 03 stage (ingress/traefik):
```shell
[terraform]: load_balancer_address = {
[terraform]:   "hostname" = "xxx.xx.x.xxx"
[terraform]:   "ip" = ""
[terraform]: }
```

Author: I think the latest push is clearer on this. lmk if it makes sense
